linux/keyman-config/keyman_config/kvk2ldml.py

```diff
@@ -349,7 +352,17 @@ def convert_ldml(kvkData):
             else:
                 keymaps["shift"] = (uskey,)
 
+    info, system, options, keyboards, files = parsemetadata(kmpJsonFilename)
+
     ldml = etree.Element("keyboard", locale="zzz-keyman")
+    for keyboard in keyboards:
+        if keyboard['id'] != keyboardName:
+            continue
+        if 'oskFont' in keyboard:
+            font, ext = os.path.splitext(keyboard['oskFont'])
+            ldml.set('keymanFacename', font)
```

Comment:
The `oskFont` property is a filename, not a face name. We sadly will need to load the .ttf to get the face name out

Reply:
done